app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -9,13 +9,11 @@ import androidx.navigation.ui.NavigationUI
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
 
-//import edu.uc.hickmadc.customnotifications.ui.main.MainFragment
-
 class MainActivity : AppCompatActivity() {
 
     //temp data
-    val language = arrayOf<String>("Brush Teeth","Eat Breakfast","Do schoolwork")
-    val description = arrayOf<String>(
+    private val language = arrayOf<String>("Brush Teeth","Eat Breakfast","Do schoolwork")
+    private val description = arrayOf<String>(
```

Comment:
Changed these values to be private, so they can't be changed outside of this class. Since this is temporary data, it shouldn't be a huge problem, but both Android Studio and the internet suggest that these arraylists should be private.
Source: https://softwareengineering.stackexchange.com/questions/143736/why-do-we-need-private-variables and https://docs.sonarqube.org/display/SONARQUBE45/Bad+Distribution+of+Complexity

Reply:
That makes sense.  Great citation.